<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>165</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>465</width>
    <height>165</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="clipIdLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Clip Id 9023490 093240904</string>
   </property>
  </widget>
  <widget class="QLabel" name="mimeTypeLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>MimeType</string>
   </property>
  </widget>
  <widget class="QLabel" name="dateLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>DateLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="userNameLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>UserNameLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Clip ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>MIME Type:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Timestamp:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>User Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>File Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="fileNameLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>FileNameLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="copyButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
